cmd/root.go

```diff
@@ -65,7 +65,7 @@ func init() {
 	// Here you will define your flags and configuration settings.
 	// Cobra supports persistent flags, which, if defined here,
 	// will be global for your application.
-	rootCmd.PersistentFlags().StringP(argRootServer, "", "https://hosted.mender.io", "root server URL, e.g. 'https://hosted.mender.io' (default)")
+	rootCmd.PersistentFlags().StringP(argRootServer, "", "https://hosted.mender.io", "root server URL, e.g. 'https://hosted.mender.io'")
```

Comment:
Question:
Not really related to the commit message, no?

Reply:
You are right, but useless. I saw it and changed it.

Reply:
No, the "(default") as is was shwon double

Reply:
Only to have mentioned it again explicitly:
```
$ ./mender-cli --help

...
Flags:
  -h, --help            help for mender-cli
      --server string   root server URL, e.g. 'https://hosted.mender.io (default)' (default "https://hosted.mender.io")
```